src/drivers/dshot/DShot.cpp

```diff
@@ -400,11 +380,6 @@ int DShot::request_esc_info()
 	return motor_index;
 }
 
-void DShot::mixerChanged()
```

Comment:
This callback is still required. We can rename it though.

Reply:
Fixed.